### PR TITLE
Feature/header footer search

### PR DESCRIPTION
**Pull request by martynaspuskunigis**

### CODE REVIEW

File: src/app/components/header/header-view.scss [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+        position: relative;
+        width: 100%;
+        height: 70;
+        background: url("../../../assets/search-banner-background.jpg") center center no-repeat;
+        background-position: 0 -130px;
+        background-size: cover;
+        display: flex;
+        justify-content: center;
+
+        // &:before {
```
Comment (EimD) [inline src/app/components/header/header-view.scss]: Remove unnecessary comments.

File: src/app/components/header/header-view.scss [hunk 2]
```diff
@@ -0,0 +1,47 @@
+.header-view {
+    display: flex;
+    justify-content: center;
+    .navigation {
```
Comment (EimD) [inline src/app/components/header/header-view.scss]: Add empty line before `.navigation`.

File: src/app/components/router/router.tsx [hunk 3]
```diff
@@ -2,18 +2,28 @@ import * as React from "react";
...
 export class Router extends React.Component {
     public render(): JSX.Element | JSX.Element[] {
         return (
             <BrowserRouter>
-                <Switch>
-                    <Route path="/" component={HomeView} exact />
-                    <Route path="/recipe/:id" component={ActiveRecipeContainer} />
-                    <Route path="/favorites" component={FavRecipesView} />
-                </Switch>
+                <div className="router">
```
Comment (EimD) [inline src/app/components/router/router.tsx]: Make base layout with `this.props.children`.

File: src/app/components/footer/footer-view.tsx [hunk 4]
```diff
@@ -0,0 +1,15 @@
+import * as React from "react";
+
+import "./footer-view.css";
+
+export class FooterView extends React.Component {
+    public render(): JSX.Element {
+        return (
+            <div className="footer-view">
+                <div className="wrapper">
+                    <div className="footer-text">© 2018</div>
```
Comment (DeividasBakanas) [inline src/app/components/footer/footer-view.tsx]: Make years dynamic. Use current year from current date.

File: src/app/components/header/header-view.scss [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+    display: flex;
+    justify-content: center;
+    
+    .navigation {
+        border-bottom: 2px solid white;
+        position: relative;
+        width: 100%;
+        height: 70;
+        background: url("../../../assets/search-banner-background.jpg") center center no-repeat;
+        background-position: 0 -130px;
```
Comment (DeividasBakanas) [inline src/app/components/header/header-view.scss]: Are there no issues with responsiveness?

File: src/app/components/router/content-layout.scss [hunk 6]
```diff
@@ -0,0 +1,3 @@
+.content-layout {
+    min-height: calc(100vh - 105px);
```
Comment (DeividasBakanas) [inline src/app/components/router/content-layout.scss]: What does `105px` represents? Maybe you should have a separate variable for it.

File: src/app/components/header/header-view.scss (hunk 5 above)
Comment (martynaspuskunigis) [reply src/app/components/header/header-view.scss]: It works as intended.

**Pull request merged**